test/controllers/api_users/permissions_controller_test.rb

```diff
@@ -2,99 +2,103 @@
 
 class ApiUsers::PermissionsControllerTest < ActionController::TestCase
   context "#edit" do
-    should "prevent unauthenticated users" do
```

Comment:
The diff here ("Group related tests in contexts ") is a bit horrible I think because of the move + context nesting in one go - would it be possible to reorder then nest in two separate commits, at least temporarily?